Tests/Unit/InMemoryAccessTokenManagerTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Trikoder\Bundle\OAuth2Bundle\Tests\Unit;
+
+use DateTime;
+use PHPUnit\Framework\TestCase;
+use ReflectionProperty;
+use Trikoder\Bundle\OAuth2Bundle\Manager\InMemory\AccessTokenManager as InMemoryAccessTokenManager;
+use Trikoder\Bundle\OAuth2Bundle\Model\AccessToken;
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+
+final class InMemoryAccessTokenManagerTest extends TestCase
+{
+    public function testClearExpired(): void
+    {
+        $inMemoryAccessTokenManager = new InMemoryAccessTokenManager();
+
+        timecop_freeze(new DateTime());
+
+        $testData = $this->buildClearExpiredTestData();
+
+        foreach ($testData['input'] as $token) {
+            $inMemoryAccessTokenManager->save($token);
+        }
+        $this->assertSame(3, $inMemoryAccessTokenManager->clearExpired());
+
+        timecop_return();
```

Comment:
@HypeMC Please use the setUp/tearDown methods for (un)freezing time or wrap this chunk of the code in a try {} finally {} block. The way the test is currently written if the assertion above the `timecop_return` call fails an exception will be thrown which means that `timecop_return` won't be called and all the rest of the tests will be run with frozen time. The same applies for the `InMemoryRefreshTokenManagerTest` too.

Reply:
@X-Coder264 Agreed, there are several tests that use timecop incorrectly, I'll create a separate PR to fix all of them at once, so I'd leave this as is for now.